spec/support/split_checkout_helper.rb

```diff
@@ -46,4 +42,14 @@ def proceed_to_payment
     click_button "Next - Payment method"
     expect(page).to have_button("Next - Order summary")
   end
+
+  def proceed_to_summary
+    click_on "Next - Order summary"
+    expect(page).to have_content "Shopping @ #{distributor.name}"
```

Comment:
This helper assumes that the calling test has `distributor` defined as method, e.g. with `let`. That can be confusing when you try to use this helper in a different test and call your distributor method differently.

Reply:
Absolutely, I've stumbled upon that issue before. I've changed it into an assertion to find the "Complete order" button :+1: